### DISCUSSION TITLE
Video and audio not showing on my website

### DISCUSSION THREAD

**Discussion by collinxander** [community/community#127721] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why isnt my videos and audio not showing on my website when i input the code correctly and i have the files on my repository.
<img width="1440" alt="Screenshot 2024-06-07 at 9 52 37 AM" src="https://github.com/community/community/assets/120601224/8b0c61bd-26da-47cf-b05b-4e5d382c2713">
<img width="1440" alt="Screenshot 2024-06-07 at 9 52 49 AM" src="https://github.com/community/community/assets/120601224/21b31da3-f874-4b98-b2ce-1899e16da1c3">
<img width="1440" alt="Screenshot 2024-06-07 at 9 52 59 AM" src="https://github.com/community/community/assets/120601224/53471966-425f-486b-b776-711451f9a82b">
<img width="1440" alt="Screenshot 2024-06-07 at 9 57 35 AM" src="https://github.com/community/community/assets/120601224/f09b802c-0be7-433b-94cd-10cc1a851aef">

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @collinxander , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
